test/unit/component/parser/json/BpmnJsonParser.event.boundary.non.interrupting.timer.test.ts

```diff
@@ -0,0 +1,286 @@
+
+    verifyShape(model.flowNodes[1], {
+      shapeId: 'shape_boundaryEvent_id_0',
+      parentId: 'task_id_0',
+      bpmnElementId: 'event_id_0',
+      bpmnElementName: 'event name',
+      bpmnElementKind: ShapeBpmnElementKind.EVENT_BOUNDARY,
+      bounds: {
+        x: 362,
+        y: 232,
+        width: 36,
+        height: 45,
+      },
+    });
+  });
+
+  it('json containing one process with a timer non-interrupting boundary event, attached to an activity, without cancelActivity attribute, timer non-interrupting boundary event is present', () => {
+    const json = `{
+                "definitions" : {
+                    "process": {
+                        "task": {
+                            "id":"task_id_0",
+                            "name":"task name"
+                        },
+                        "boundaryEvent": {
+                            "id":"event_id_0",
+                            "name":"event name",
+                            "attachedToRef":"task_id_0",
+                            "timerEventDefinition": ""
+                        }
+                    },
+                    "BPMNDiagram": {
+                        "name":"process 0",
+                        "BPMNPlane": {
+                            "BPMNShape": [
+                              {
+                                  "id":"shape_task_id_0",
+                                  "bpmnElement":"task_id_0",
+                                  "Bounds": { "x": 362, "y": 232, "width": 36, "height": 45 }
+                              },
+                              {
+                                  "id":"shape_boundaryEvent_id_0",
+                                  "bpmnElement":"event_id_0",
+                                  "Bounds": { "x": 362, "y": 232, "width": 36, "height": 45 }
+                              }
+                            ]
+                        }
+                    }
+                }
+            }`;
+
+    const model = parseJsonAndExpectOnlyBoundaryEvent(json, ShapeBpmnEventKind.TIMER, 1, true);
+
+    verifyShape(model.flowNodes[1], {
+      shapeId: 'shape_boundaryEvent_id_0',
+      parentId: 'task_id_0',
+      bpmnElementId: 'event_id_0',
+      bpmnElementName: 'event name',
+      bpmnElementKind: ShapeBpmnElementKind.EVENT_BOUNDARY,
+      bounds: {
+        x: 362,
+        y: 232,
+        width: 36,
+        height: 45,
+      },
+    });
+  });
+
+  it('json containing one process with a non-interrupting boundary event, attached to an activity, with timer definition and another definition, timer event is NOT present', () => {
+    const json = `{
+    "definitions" : {
+        "process": {
+            "task": { "id":"task_id_0", "name":"task name" },
+            "boundaryEvent": { 
+                "id":"event_id_0", 
+                "attachedToRef":"task_id_0",
+                "cancelActivity":true,
+                "timerEventDefinition": "", 
```

Comment:
:warning: duplicate timerEventDefinition, should be something else

Reply:
Done